v03_pipeline/lib/annotations/shared.py

```diff
@@ -8,6 +8,103 @@
 
 from v03_pipeline.lib.model import ReferenceGenome
 
+AMINO_ACIDS = [
+    'A',
+    'C',
+    'D',
+    'E',
+    'F',
+    'G',
+    'H',
+    'I',
+    'K',
+    'L',
+    'M',
+    'N',
+    'P',
+    'Q',
+    'R',
+    'S',
+    'T',
+    'V',
+    'W',
+    'Y',
+    'X',
+    '*',
+    'U',
+]
+AMINO_ACIDS_LOOKUP = hl.dict(hl.enumerate(AMINO_ACIDS, index_first=False))
+
+BIOTYPES = [
```

Comment:
Sourced from https://www.gencodegenes.org/pages/biotypes.html

The formats line up at least.